frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -192,29 +198,24 @@ public void run(ProgressIndicator indicator) {
 
           GitCommandResult result = Git.getInstance().runCommand(resetHandler);
 
-          // git4idea methods are used instead of our RefreshStatusAction, to refresh state not only in our plugin,
-          // but also in whole IDE. Repository status will be refreshed also in our plugin because we are subscribed
-          // to IDE event bus.
-          refreshRepo(project, gitRepository);
+          if (result.success()) {
+            VcsNotifier.getInstance(project)
+                .notifySuccess(
+                    format(getString("action.GitMachete.BaseResetBranchToRemoteAction.notification.success"), branchName));
+            log().debug(() -> "Branch '${branchName}' reset to its remote tracking branch");
```

Comment:
Nit: `has been reset` for clarity since present tense & past participle are the same

Reply:
here and within notification!